### DISCUSSION TITLE
Different RSSI displayed: sub 1GHz or wifi?

### DISCUSSION THREAD

**Discussion by SuperNinja73** [tbnobody/OpenDTU#1580] [Q&A]
I have an HMS1000 so a cmt2300A 865MHz which works perfectly.
in the console I see:
```
13:52:57.527 > Fetch inverter: 1144xxxxxxxxxxx
13:52:57.631 > TX RealTimeRunData 865.00 MHz --> 15 85 00 12 12 80 15 19 60 80 0B 00 65 81 92 26 00 00 00 00 00 00 00 00 AE 95 9C
13:52:57.730 > Interrupt received
13:52:57.778 > RX 865.00 MHz --> 95 85 00 12 12 80 15 19 60 01 00 01 01 93 01 95 03 CB 03 C8 0F 51 0F 52 00 00 FA | -64 dBm
13:52:57.934 > Interrupt received
13:52:58.038 > RX 865.00 MHz --> 95 85 00 12 12 80 15 19 60 02 10 E1 00 00 11 36 03 0A 03 7D 09 44 13 89 1C F4 60 | -64 dBm
13:52:58.099 > Interrupt received
13:52:58.156 > RX 865.00 MHz --> 95 85 00 12 12 80 15 19 60 83 00 06 01 38 03 E7 01 AB 00 02 7F B7 C4 | -64 dBm
13:52:58.243 > RX Period End
13:52:58.243 > Success
```

and if I subscribe to the topic: `solar/dtu/rssi`: I have
`-54`

My question: who owns these 2 different RSSI s? WiFi ? CMT2300A?

**Comment by tbnobody** [comment]
The value which is sent via MQTT is the Wifi RSSI (it's in the DTU topic, not  the inverter topic). It should match up  the values show in Info --> Network Information

Inverter RSSI is (not yet) published. (One reason is, that the nRF modules don't provide a detailed RSSI information)

**Comment by SuperNinja73** [comment]
> Wifi RSSI (it's in the DTU topic, not the inverter topic)

OK I understand : when I refresh `Info --> Network Information`, RSSI change -52...-49...-63 that's why I don't have the same values in the Console and in the dtu/rssi MQTT topic !

> Inverter RSSI is (not yet) published. (One reason is, that the nRF modules don't provide a detailed RSSI information)

>  I have an HMS1000 so a cmt2300A

it seems to me that RSSI is supported in E49-900M20S:
https://github.com/lumapu/ahoy/wiki/RF-Module-CMT2300A---E49-900M20S
